redbot/cogs/audio/converters.py

```diff
@@ -405,33 +334,18 @@ class ComplexScopeParser(commands.Converter):
             source_user = None
             from_user_raw = " ".join(from_author).strip()
             if from_user_raw.isnumeric():
-                from_user_raw = int(from_user_raw)
-                try:
-                    target_user = ctx.bot.get_user(from_user_raw)
-                except Exception:
-                    source_user = None
-                from_user_raw = str(from_user_raw)
+                source_user = ctx.bot.get_user(int(from_user_raw))
```

Comment:
Same issue with `½` as before.

Reply:
Handled this (and other) by changing the pre-condition to `.isdigit` from `.isnumeric` as isdigit is both strict enough here, and matches anything we would want to guess might be a snowflake.